proseco/core.py

```diff
@@ -247,6 +245,9 @@ def __add__(self, other):
     def __radd__(self, other):
         return self.__add__(other)
 
+    def __hash__(self):
+        return hash((self.y, self.z))
```

Comment:
Though I just ran coverage and it looks like this isn't hit in any test.

Reply:
I did test this once from the command line. Since it is not actually used anywhere I didn't add a test.